print-babel.c

```diff
@@ -282,12 +297,11 @@ subtlvs_print(netdissect_options *ndo,
             ND_PRINT(" sub-pad1");
             continue;
         }
-        if(cp == ep)
-            goto invalid;
+        if (cp == ep) goto invalid;
         sublen = GET_U_1(cp);
         cp++;
-        if(cp + sublen > ep)
-            goto invalid;
+        if ((MANDATORY_MASK & subtype) != 0) ND_PRINT(" (M)");
+        if(cp + sublen > ep) goto invalid;
```

Comment:
This line should probably be a TCHECK macro.

Reply:
Looks like yes, there are some new pcap files, thank you.

Reply:
Thank you for your review. 
About the TCHECK macro, I don't see how we can use it. The only one which is suitable for is `ND_TCHECK_LEN` but it goes to trunc label (and not invalid label). Should we write a new one in netdissect.h or let the code like this ?

Reply:
The ND_TTEST_ macros do the same tests as do the ND_TCHECK_ macros, but return a Boolean result (true if the test succeeds, false if the test fails), rather than testing the Boolean result and jumping to `trunc` if the test fails.

And the GET_ macros have the tests built in.